libs/s25main/SnapOffset.h

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2005 - 2023 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Point.h"
+
+/// Type used for specifying how much a window has to be moved to snap to another window's edge. Signed base type!
+using SnapOffset = Position;
```

Comment:
I would have been explicit: `Point<int>` but I guess that is equally correct/confusing as it's neither a point nor a position :shrug:

Reply:
I've simply copied it from `DrawPoint.h`. I think this conveys/preserves compatibility with `DrawPoint`, which is the other `Point` type that `SnapOffset` interacts with frequently.